docs/source/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -109,7 +160,38 @@ For the `DIS5K <https://xuebinqin.github.io/dis/index.html>`_ we prepared random
 Self-supervised Learning
 ************************
 
-To be added soon
+Self-supervised learning can be one of the solutions if the user has a small data set, but label information is not yet available.
+General self-supervised Learning in academia is commonly used to obtain well-pretrained weights from a source dataset without label information.
+However, in real-world industries, it is difficult to apply because of small datasets, limited resources, or training in minutes.
+
+For these cases, OTX provides improved self-supervised learning recipes that can be applied to the above harsh environments.
+We adapted `DetCon <https://arxiv.org/abs/2103.10957>`_ as our self-supervised method.
+It takes some time to use these self-supervised learning recipes, but you can expect improved performance, especially in small-data regimes.
+
+The below table shows how much performance (mDice) self-supervised methods improved compared with baseline performance on the subsets of Pascal VOC 2012 with three classes (person, car, bicycle).
+The number of the subset datasets is 8, 16, and 24.
+
+`three classes (person, car, bicycle)`
+
++--------------------+-------+---------+-------+---------+-------+---------+
+| Model name         | #8    |         | #16   |         | #24   |         |
++====================+=======+=========+=======+=========+=======+=========+
+|                    | SL    | Self-SL | SL    | Self-SL | SL    | Self-SL |
++--------------------+-------+---------+-------+---------+-------+---------+
+| Lite-HRNet-s-mod2  | 48.30 | 53.55   | 57.08 | 58.96   | 62.40 | 63.46   |
++--------------------+-------+---------+-------+---------+-------+---------+
+| Lite-HRNet-18-mod2 | 53.47 | 49.20   | 56.69 | 58.72   | 62.81 | 63.63   |
++--------------------+-------+---------+-------+---------+-------+---------+
+| Lite-HRNet-x-mod3  | 50.23 | 50.93   | 60.09 | 61.61   | 62.66 | 64.87   |
++--------------------+-------+---------+-------+---------+-------+---------+
+
+You can use Self-supervised learning like the below command.
```

Comment:
To enable Self-supervised training the command below can be executed:


Anyway, here we need some concrete example, bacause it is different training type. What should we pass to the data roots? For me, as from the user perspective - it is confusing. Please, describe more about the datasets, data roots and give a bit more concrete example how to launch this training type

Reply:
Done